packages/sanity/src/core/releases/store/createReleaseOperationStore.ts

```diff
@@ -64,41 +64,38 @@ export function createReleaseOperationsStore(options: {
       },
     ])
 
-  const handleScheduleRelease = async (releaseId: string, publishAt: Date) => {
-    await requestAction(client, [
+  const handleScheduleRelease = async (releaseId: string, publishAt: Date) =>
```

Comment:
`async` can be removed from here and the other functions now that there's no await (saves us a microtask), but would be good if we still can make it return a void promise